canvg.js

```diff
@@ -2915,8 +2915,10 @@
 			// render loop
 			var isFirstRender = true;
 			var draw = function() {
-				svg.ViewPort.Clear();
-				if (ctx.canvas.parentNode) svg.ViewPort.SetCurrent(ctx.canvas.parentNode.clientWidth, ctx.canvas.parentNode.clientHeight);
+				if (isFirstRender) {

```

Comment:
why is this here? can we fix it in another way?  this variable is meant for actual first rendering not to solve browser issues and we should make the if condition target browser features